app/templates/about.hbs

```diff
@@ -1,99 +1,198 @@
 <section class="container content-section autojustify" style="margin-bottom:40px;">
-    <h2 id="about">O KSI</h2>
-    <p>KSI, neboli Korespondenční seminář z&nbsp;informatiky, je celoroční soutěž organizovaná studenty Fakulty informatiky Masarykovy univerzity. Cílem semináře je seznámit řešitele se zajímavými oblastmi informatiky a&nbsp;procvičit programátorské, matematické a&nbsp;logické myšlení.
-    Seminář je uzpůsoben jak pro úplné začátečníky, kteří si na jednoduchých příkladech procvičí danou problematiku, tak pro zkušenější řešitele, kteří se pokusí pokořit hlavní soutěžní úlohy.
+ <h2 id="about">O KSI</h2>
+  <p>
+  KSI, neboli Korespondenční seminář z informatiky, je celoroční soutěž

```

Comment:
Přímo v HTML nemůžeš entitu `&nbsp;` zaměnit za znak pevné mezery. Jsou to dva různé znaky/entity a mají různé chování.

V markdownu jsem byl ochoten zavřít oči a doufat, že to Pandoc přeloží správně, ale tady k překladu (v podstatě) nedochází.

_(zaměnil jsi jich víc, připisuji to jenom k této)_

Reply:
Já jsem si to zkoušel a minimálně ve FF se mi `&nbsp;` a UTF znak nedělitelné mezery chovají stejně. Jaký je rozdíl?

Reply:
Obojí funguje jako nedělitelná mezera, tu vlastnost to splňuje.

V čem se ale liší je například zda `<span>x</span>`, kde _x_ je jedna ze zmíněných mezer, bude prázdný div s nulovou šířkou a výškou (znak pevné mezery), nebo jednoznakový který se nezmenší.
_chování pro znak pevné mezery se navíc liší (nebo alespoň lišilo) skrz prohlížeče_

Stejný rozdíl se pak projevuje, když s takovými elementy zachází CSS a JS. Pro `&nbsp;` je zalamování třeba v jQuery ošetřené, pro pevnou mezeru nebývalo (možná už je, nevím).

Reply:
PS: Otestoval jsem si podporu UTF znaku pevné mezery a vypadá to, že JS knihovny se již už naučili ho správně zpracovat. A jelikož si ve stránce nikde nehrajeme s prázdnými elementy, nechám je tam jako UTF znaky.

Reply:
Ok.